Source/WebKit/UIProcess/WebPageProxy.h

```diff
@@ -3471,6 +3473,8 @@ class WebPageProxy final : public API::ObjectImpl<API::Object::Type::Page>, publ
 
     RefPtr<WebPageProxy> m_pageToCloneSessionStorageFrom;
     Ref<BrowsingContextGroup> m_browsingContextGroup;
+
+    std::optional<WebsitePoliciesData> m_mainFrameWebsitePoliciesData;
```

Comment:
Let's make this a unique_ptr.  Then we don't need to include WebsitePoliciesData.h.  If we really do want to use std::optional, then it should go in WebPageProxyInternals.h.